.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xcode/SDL"]
+	path = xcode/SDL
+	url = https://github.com/mattuna15/SDL.git
```

Comment:
This is a little scary.
* Can this be done without a submodule? Something like: For an iPad build, manually drop SDL.zip into this directory? Sure, hacky, but a submodule is quite a big hammer, and it's not needed for the other platforms.
* I see your SDL fork has a few changes. Why are they needed, and can we make this work with a vanilla SDL?

Reply:
Sure I can do that.

Reply:
I need to look at subclassing the SDL code in the main app.

Reply:
sub module removed

Reply:
Code changed now to use vanilla SDL included as a zip :)